### PR TITLE
Create Nav Bar for the welcome page

### PR DESCRIPTION
**Pull request by jfewii**

### CODE REVIEW

File: postal-injection/src/main/python/GeoPy.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Calculate distances using GeoPy
+from geopy.distance import geodesic
+
+kolkata = (22.5726, 88.3639) 
+delhi = (28.7041, 77.1025)
+
+print(geodesic(kolkata, delhi).km)
```
Comment (jennygaoo) [inline postal-injection/src/main/python/GeoPy.py:7]: nit: add empty line after line 7

File: postal-injection/src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <div class="w3-right w3-hide-small">
+          <a href="#about" class="w3-bar-item w3-button">ABOUT</a>
+          <a href="#work" class="w3-bar-item w3-button"><i class="fa fa-th"></i>PRODUCT</a>
+          <a href="#team" class="w3-bar-item w3-button"><i class="fa fa-user"></i> TEAM</a>
+          <a href="#contact" class="w3-bar-item w3-button"><i class="fa fa-envelope"></i> CONTACT</a>
+        </div>
+      </div>
+    </div>
+  </body>       
+</html>
```
Comment (jennygaoo) [inline postal-injection/src/main/webapp/index.html:28]: same as last comment

File: postal-injection/src/main/webapp/style.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  height: 70%;  
+}
+
+/* Full height image header */
+.bgimg-1 {
+  background-position: center;
+  background-size: cover;
+  background-image: url("/images/home.jpg");
+  min-height: 100%;
+}
```
Comment (jennygaoo) [inline postal-injection/src/main/webapp/style.css:25]: same as last comment

**Comment by jennygaoo** [comment]
if you haven't already, could you pull from master? the .gitignore in master will help us avoid this merge conflict

**Comment by jennygaoo** [comment]
also, I ran this in JSFiddle and it looks good! :)

File: postal-injection/target/postal-injection-1/index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        </div>
+        <!-- Hide right-floated links on small screens and replace them with a menu icon -->
+        <a href="javascript:void(0)" class="w3-bar-item w3-button w3-right w3-hide-large
+          w3-hide-medium" onclick="w3_open()">
+          <i class="fa fa-bars"></i>
+        </a>
+      </div>
+    </div>  
+  </body>       
+</html>
```
Comment (andreahu12) [inline postal-injection/target/postal-injection-1/index.html:32]: nit: newline needed here as well

File: postal-injection/src/main/python/GeoPy.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#Calculate distances using GeoPy
```
Comment (andreahu12) [inline postal-injection/src/main/python/GeoPy.py:1]: shouldn't this change be in https://github.com/googleinterns/step8-2020/pull/3?

**Comment by andreahu12** [comment]
> also, I ran this in JSFiddle and it looks good! :)

+1 nice design

File: postal-injection/src/main/python/GeoPy.py (hunk 5 above)
Comment (jfewii) [reply postal-injection/src/main/python/GeoPy.py:1]: I'm not sure how it got into this pr because I fixed the reviews in #3 already. I believe it had something to do with gitignore.

File: postal-injection/src/main/python/GeoPy.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
```
Comment (andreahu12) [inline postal-injection/src/main/python/GeoPy.py:1]: This file is unrelated and should be in a separate PR

File: postal-injection/target/postal-injection-1/index.html (hunk 4 above)
Comment (jennygaoo) [reply postal-injection/target/postal-injection-1/index.html:32]: bump ^

File: postal-injection/src/main/python/GeoPy.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
+# Calculate distances using GeoPy
+# Load the Pandas libraries with alias 'pd' 
+from geopy.distance import geodesic
+import pandas as pd 
+
+df = pd.read_csv("shipData3.csv")
```
Comment (jgao23) [inline postal-injection/src/main/python/GeoPy.py:6]: supernit: typically w/ python scripts, you'll want to define a main() function and call it like so:
``` if __name__ == "__main__":
    main()
```
https://realpython.com/python-main-function/
